.circleci/config.yml

```diff
@@ -6,112 +6,44 @@ jobs:
       - image: circleci/node:8
     steps:
       - checkout
-    
-      - restore_cache:
-          name: Restore node modules cache
-          key: lock-v1-{{ checksum "package-lock.json" }}-{{ arch }}
-
-      - run: npm install 
-
-      - save_cache:
-          name: Save node modules cache
-          key: lock-v1-{{ checksum "package-lock.json" }}-{{ arch }}
-          paths:
-            - node_modules
-
+      - run: npm install
       - run:
-          name: jest tests
+          name: Test
           command: |
             mkdir -p test-results/jest
-            npm run test-js
+            npm run test
           environment:
             JEST_JUNIT_OUTPUT: test-results/jest/junit.xml
-
       - persist_to_workspace:
           root: ~/react-native-radar
           paths:
             - node_modules
-
       - store_test_results:
           path: test-results
-
       - store_artifacts:
           path: test-results
-
   android:
-    working_directory: ~/react-native-radar
+    working_directory: ~/react-native-radar/example
     docker:
       - image: circleci/android:api-28-node
     steps:
       - checkout:
-
-      - attach_workspace:
-          at: ~/react-native-radar
-      
-      - run: npm run test-android
-
-      - run:
-          name: Save test results
-          command: |
-            mkdir -p test-results/android/
-            find ./android -type f -regex ".*/build/test-results/.*xml" -exec cp {} ./test-results/android/ \;
-          when: always
-
-      - store_test_results:
-          path: test-results
-
-      - store_artifacts:
-          path: test-results
-
+          path: ~/react-native-radar
+      - run: yarn install
+      - run: sudo npm install -g react-native
+      - run: chown -R $USER:$USER android
+      - run: cd android && ./gradlew assembleDebug
   ios:
-    macos: 
-      xcode: "11.0.0"
-    working_directory: ~/react-native-radar
-    
+    macos:
+      xcode: "11.2.1"
+    working_directory: ~/react-native-radar/example
     steps:
-      - checkout
-
-      - restore_cache:
-          name: Restore node modules cache
-          key: lock-v1-{{ checksum "package-lock.json" }}-{{ arch }}
-
-      - run: npm install
- 
-      - save_cache:
```

Comment:
do we not want to cache installed modules here? I guess we can always add this back later

Reply:
I was running into weirdness with caching, will see if I can add it back

Reply:
Challenge with caching here is that the example app pulls the module code from the parent directory instead of npm. We want it to pull a fresh version every time, caching messed with that. Could probably get smarter about this but this doesn't take terribly long to run, so I think we're fine for now.

Reply:
I see, we can probably check the ts or checksum on the parent directory to see if a fresh pull is needed - agreed not a big deal for this iteration